app/models/miq_ae_domain.rb

```diff
@@ -212,8 +216,8 @@ def squeeze_priorities
   end
 
   def about_class
-    ns = children.find_by("lower(name) = ?", "system")
-    MiqAeClass.where(:namespace_id => ns.id).find_by("lower(name) = ?", "about") if ns
+    ns = ae_namespaces.where("lower(name) = ?", "system").select(:id)
+    MiqAeClass.where(:namespace_id => ns).find_by("lower(name) = ?", "about")
```

Comment:
Since we have a much better lookup now, can we use it here and just lookup by "System/About" ?